tests/ecosystem/upgrade/test_monitoring_after_ocp_upgrade.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
plz add blank line between 3rd party and built-in modules import

Reply:
done